config/locales/en.yml

```diff
@@ -157,13 +157,14 @@ en:
           href: "/browse/visas-immigration"
         - label: Working, jobs and pensions
           href: "/browse/working"
-      - label: Departments
-        href: "/government/organisations"
       - label: Government activity
         href: "/search/news-and-communications"
-        description: Find out what the government is doing
+        description: Search for a department and find out what the government is doing
         menu_contents: # If adding or removing items, remember to update the
                        # `columns` in the layout-super-navigation-header SCSS.
+        - label: Departments
+          href: "/government/organisations"
+          description: Departments, agencies and public bodies
```

Comment:
Note to self - check that the tracking has been updated for this repositioned link.

Reply:
Yep, it's fine; output is:

```html
<a 
  class="govuk-link gem-c-layout-super-navigation-header__navigation-second-item-link gem-c-layout-super-navigation-header__navigation-second-item-link--with-description" 
  data-module="gem-track-click" 
  data-track-action="governmentactivityLink" 
  data-track-category="headerClicked" 
  data-track-label="/government/organisations" 
  data-track-dimension="Departments" 
  data-track-dimension-index="29" 
  href="/government/organisations">
    Departments
</a>
```